### PR TITLE
documentation updated with icon import and cinnabar/zinc

### PR DESCRIPTION
**Pull request by sensescape**
Holding for blog post and CDN publication... ETA: ???

Connects to #33 and #34.

- [x] @nvkelso to remove Zinc from the icon table.

### CODE REVIEW

File: icons.md [hunk 1]
```diff
@@ -21,11 +21,35 @@ layers:
     my-layer:
         data: { source: my-source }
         draw:
-            icons:
+            mapzen_icon_library:
```
Comment (nvkelso) [inline icons.md]: We should have two examples, one that is:

```
         draw:
            mapzen_icon_library:
               sprite: zoo
```

Second that says if your data has a kind property with one of the values in the table below, it'll just show up:

```
         draw:
            mapzen_icon_library: {}
```

File: icons.md [hunk 2]
```diff
@@ -361,47 +385,14 @@ winter_sports | ![winter_sports](img/sprite/bubble-wrap-style/2x/winter_sports.p
...
 While earlier versions of Mapzen basemap styles supported _most_ icons in the table above, the following versions added full support:
 
-* Bubble Wrap: `6.0.0`
-* Walkabout: `3.0.0`
-* Tron: `3.0.0`
-* Refill: `6.0.0`
-* Cinnabar: `6.0.0`
-* Zinc: `5.0.0`
\ No newline at end of file
+* Bubble Wrap: `7.0.0 and later`
```
Comment (nvkelso) [inline icons.md]: Keep at the original number (back to 6), same for below...

File: styles.md [hunk 3]
```diff
@@ -106,42 +106,24 @@ Current **MAJOR** versioned release (includes any minor and patch updates):
 * `https://mapzen.com/carto/tron-style-more-labels/tron-style-more-labels.zip`
 * `https://mapzen.com/carto/tron-style-no-labels/tron-style-no-labels.zip`
 
-## Cinnabar
+## Cinnabar and Zinc
```
Comment (nvkelso) [inline styles.md]: This would just be `Cinnabar`.

File: themes.md [hunk 4]
```diff
@@ -152,3 +152,55 @@ Current **MAJOR** versioned releases:
...
 | 11     |           | `https://mapzen.com/carto/refill-style/8/themes/detail-11.zip`
+
+
+## Cinnabar
+
+Cinnabar Style supports 2 different theme options: **color** and **label**.
+
+### Color
+
+Zinc is now a color theme of Cinnabar.
```
Comment (nvkelso) [inline themes.md]: To: `Cinnabar is offered in one additional color theme: Zinc.`

File: icons.md (hunk 1 above)
Comment (nvkelso) [reply icons.md]: Ah, the top example in full:

```
import:
    - https://mapzen.com/carto/refill-style/refill-style.zip

sources:
    my-source:
        type: GeoJSON
        url:  https://example.com/filename.geojson
        max_zoom: 16

layers:
    my-layer:
        data:
            source: my-source
        draw:
            mapzen_icon_library:
                sprite: zoo
            # if your data includes a kind property with values in the table below
            # the sprite will be sourced per feature to the appropriate icon
            # mapzen_icon_library: {}
```

File: icons.md [hunk 5]
```diff
@@ -21,11 +21,35 @@ layers:
...
+You can use the icons in this library independent of one of the Mapzen basemap styles, but there are a few required steps to doing this in Tangram:
+
+- You'll need to setup your own textures block with the spritesheet, and named sprites block.
+- You'll need to create a new style based on points, referencing the textures
+- In each layer's draw block, include the style:
+
+```
+import:
+    - https://mapzen.com/carto/refill-style/refill-style.zip
+    - https://mapzen.com/carto/refill-style/refill-icons.zip
```
Comment (nvkelso) [inline icons.md]: I think this whole section goes to:

```
import:
     - https://mapzen.com/carto/refill-style/refill-icons.zip
 
sources:
    my-source:
        type: GeoJSON
        url:  https://example.com/filename.geojson
        max_zoom: 16

layers:
    my-layer:
        data:
            source: my-source
        draw:
            mapzen_icon_library:
                sprite: zoo
            # if your data includes a kind property with values in the table below
            # the sprite will be sourced per feature to the appropriate icon
            # mapzen_icon_library: {}
```

File: icons.md [hunk 6]
```diff
@@ -21,11 +21,35 @@ layers:
...
         draw:
-            icons:
+            mapzen_icon_library:
                 size: [[13, 18px], [16, 18px], [18, 22px]]
                 sprite: zoo
 ```
 
+## Using Mapzen icons independently
+
+You can use the icons in this library independent of one of the Mapzen basemap styles, but there are a few required steps to doing this in Tangram:
```
Comment (nvkelso) [inline icons.md]: You can remove: `, but there are a few required steps to doing this in Tangram:`

File: icons.md [hunk 7]
```diff
@@ -21,11 +21,35 @@ layers:
...
+
+- You'll need to setup your own textures block with the spritesheet, and named sprites block.
```
Comment (nvkelso) [inline icons.md]: This list can be removed.

File: icons.md (hunk 6 above)
Comment (nvkelso) [reply icons.md]: To:

`You can use the icons in the Mapzen icon library independent of one of the Mapzen basemap styles by importing just the icon assets.`

**Comment by nvkelso** [comment]
Per IRL discussions we're moving Zinc to be a theme of Refill, which means the Zinc documentation here needs to be updated.

**Review by nvkelso — CHANGES_REQUESTED** [review]
See https://github.com/tangrams/cartography-docs/pull/35#issuecomment-328661041 to move Zinc to be a Refill theme.

**Comment by nvkelso** [comment]
We did a poor job of talking about the built-in style `icons:` in the documentation so far... https://mapzen.com/documentation/cartography/icons/. We've resolved that in this PR, but...

But `labels:` was broadcast a few times like in Rhonda's https://mapzen.com/blog/lots-of-dots/#built-in-sprites.

For backwards compatibility we could add in the styles section:

```
    icons:
        base: mapzen_icon_library
```

But then remove it in the next major version?

**Comment by rmglennon** [comment]
For migration_guide, could you give a more specific title so it's clear what people are migrating to and whether it affects them?

**Comment by nvkelso** [comment]
@rmglennon re https://github.com/tangrams/cartography-docs/pull/35#issuecomment-332311350 do you have a recommendation, please?

Another option is to nest the migration guide into the semantic versioning section.

The migration guide is similar to the `Migrating to version 1.0` vector tiles documentation. For these cartography docs I don't want to do a migration guide for every single breaking change, but for significant changes this gives us a place to give some tips.

**Comment by rmglennon** [comment]
Ah, if you intend to use this page again in the future, then you could add a heading above the Zinc with ## and then add a # to the current ones to make them nest under it. It sounds like it could be something like, "Migration to cartography version 6 (September 2017)"? 

I'm not sure how we refer to cartography releases myself, so that's why I added the comment. :)

**Comment by nvkelso** [comment]
Yeah, it's complicated as each style is independently versioned – this is partly why I added the release management issue as even I was getting confused 😉 Sometimes they are released in a batch (like this PR described), sometimes it's just a single style being updated like Refill last month. And then the docs are also versioned for "public" release points but that's wishy-washy.

The current versions are (with this "release"):

- Bubble Wrap: `8.0.0`
- Cinnabar: `8.0.0`
- Refill: `9.0.0`
- Tron: `5.0.0`
- Walkabout: `6.0.0`

There are already two items in the [migration guide](https://github.com/tangrams/cartography-docs/blob/388207618245a5a2550f592e3a929dc96ad4a080/migration-guide.md): 

- **Zinc**
- **Named label variants**. 

Should I modify the headers so they say:

- **Named label variants (September 2017)** 
- **Zinc (September 2017)** 

And sort most recent at the top, and then a-z inside the same date range (so Zinc would move down)?

We don't have release notes per say right now in the documentation site. Instead those are only maintained in the source Github repo's release notes functionality.

**Comment by rmglennon** [comment]
Whew, OK...sounds like a "use your judgement" situation on whether or not and how to identify them. :)

**Review by nvkelso — APPROVED** [review]
LGTM

**Pull request merged**